src/pages/index.vue

```diff
@@ -46,21 +46,40 @@
   import HomeSponsors from '~/components/HomeSponsors/Index.vue'
   import HomeVideo from '~/components/HomeVideo/Index.vue'
 
+  import { mapActions, mapGetters } from 'vuex'
+
```

Comment:
Probar a obtener el host, revisar:
```suggestion
this.$nuxt.$route
```